### PR TITLE
Profile side menu update

### PR DESCRIPTION
**Pull request by rostyk-kanafotskyy**
Description:

- Added order page, updated email form for b2b
 
- Added back links, title updates

- Updated dropdown menu

- Updated b2c dropdown menu
 
- update profile page style, style radio button

- Update shipping address id

- Updated tests, updated login drop down sales, moved wishlist to left … …

- Updated account link

- Updated profile tests

Linting:

- [x] No linting errors

Component Updates:

- [x] Component package requires update on npm [@elasticpath/store-components](https://www.npmjs.com/package/@elasticpath/store-components)

Tests:

- [ ] E2E tests (npm test run with `e2e`)
- [x] Manual tests

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

**Comment by shaunmaharaj** [comment]
If the store is run in b2b mode, we should also display the sidebar in the Profile page, and have a new row at the top of the sidebar for "My Account". The header of "Profile" should also be updated to "My Account".

image

**Comment by shaunmaharaj** [comment]
With the new sidebar in place, we should confirm with Bonnie if we still need all the links in this dropdown, or if these can be consolodated:

image

**Comment by shaunmaharaj** [comment]
@rostyk-kanafotskyy this looks great so far! nice job with this one!

**Comment by shaunmaharaj** [comment]
Need to address conflict in PR

File: src/components/src/AppHeaderLogin/appheaderlogin.main.tsx [hunk 1]
```diff
@@ -352,23 +384,56 @@ class AppHeaderLoginMain extends Component<AppHeaderLoginMainProps, AppHeaderLog
               </button>
               <div data-region="authMainRegion" className="auth-nav-container dropdown-menu dropdown-menu-right" aria-label="header_navbar_login_button ">
                 <ul data-el-container="global.profileMenu" className="auth-profile-menu-list">
-                  {userName !== 'undefined' ? (
-                    <li className="dropdown-header">
-                      {userName}
+                  {Config.b2b.enable ? (
+                    <li className="dropdown-item shop-for">
+                      <span className="user-name">{b2bUserName}</span>
```
Comment (shaunmaharaj) [inline src/components/src/AppHeaderLogin/appheaderlogin.main.tsx]: The value for this is displaying "lastname firstname" in the incorrect order
image

**Comment by shaunmaharaj** [comment]
Adding photos to help with UX review:
image
image

**Comment by BonnieEP** [comment]
@rostyk-kanafotskyy please see the list of tweaks needed below

Dropdown
1.  Can we bump the font weight for the links in the dropdown to:
font-weight: 400;
2. Can we add padding of 8px to the top of the dropdown frame (above name)
3. Can we re-order the info in the top

Should be in this order:
- Name
- Email
- Account (for B2B)


In Account page
1. Can we add 8px padding below all the subheaders and the content box (
Example: 'General')

Otherwise this looks great! Thank you!

**Comment by shaunmaharaj** [comment]
Thanks for the fixes @rostyk-kanafotskyy, and for the review @BonnieEP!
Did a final review and all looks good. Merging now!

**Pull request merged**